how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb

```diff
@@ -101,35 +101,13 @@
     ]
    },
    "source": [
-    "## Initialize Neptune and create new run\n",
+    "## Initialize Neptune and create a new run\n",
```

Comment:
(Commenting on this entire section) I think we can revise the init flow a bit more and bring it closer to what we worked out for the template today. We could use the Option A/B headings, for example.

Reply:
Commented before push :D